onboarding_documentation/ingestion/ingestion_tutorial.md

```diff
@@ -0,0 +1,218 @@
+
+    We will be using the bucket `cpg-sandbox-test-upload/onboarding_fastqs` for this tutorial in which a fictional collaborator has deposited data and a metadata manifest.
+
+    ![Screenshot of `cpg-sandbox`](./images/1_explore_bucket.png)
+
+2. Examine the data stored in `cpg-sandbox-test-upload/onboarding_fastqs`
+
+    Click on the link for the folder `cpg-sandbox-test-upload` and again on the link for folder `onboarding_fastqs`. You will now be located at `cpg-sandbox-test-upload/onboarding_fastqs`.
+
+    Scan down the list of files in this folder in to check manually for any inconsistencies. Alternatively, use your terminal to list your files: `gsutil ls gs://cpg-sandbox-test-upload`.
+
+    ![Screenshot of files stored in cpg-sandbox-test-upload](./images/2_investigate_data.png)
+
+    The data appear to be tidy and consistent. Note that files use the naming convention sandbox_fastq_SB_XXX_X.fastq.gz.
+
+    The majority of files are compressed fastq files, which are used as raw input files for the analysis of genomic data. A summary of genomics file formats is [available here](https://docs.google.com/presentation/d/1TQPpDjjOqFy5gVb9Tu2Qqtus4JbnNEia-jWkDe6Sm0Q/edit#slide=id.gb6ec11b86e_0_65).
+
+    A single file `onboarding_fastq_sample_file_map.csv` is a .csv file.
+
+3. Examine your metadata to ensure that they are tidy and properly representative of the data
+
+    Metadata has been provided for our tutorial by our kind fictional collaborator in the file `onboarding_fastq_sample_file_map.csv`. Metadata will not necessarily be colocated with your data and often collaborators will provide these via an alternative means such as email.
+
+    Download this file to your computer. You can do this is by clicking the download symbol on the Google Console or by using your terminal: `gsutil cp gs://cpg-sandbox-test-upload/onboarding_fastqs/onboarding_fastq_sample_file_map.csv`. Using this command will download the file to your current working directory.
+
+   ![Location of download symbol on Google Cloud](./images/3_examine_metadata.png)
+
+    Open this file using software of your choice. [Excel is ok for small-medium datasets](https://www.ibm.com/docs/en/planning-analytics/2.0.0?topic=issues-microsoft-excel-limits).
+
+    Observe that there are 20 rows of data and 4 columns. There are two rows per individual. These each have identical values for `Individual ID` and `Sample ID` but each row has a distinct value for `Filenames`. File naming uses the following structure {Sample ID}{Integer}.fastq.gz. The file extensions .fastq and .gz indicate that these are zipped FASTQ files. All data have been derived from Whole Genome Sequencing, which is summarised in the `Type` column as ‘WGS’.
+
+    The data are tidy and seem complete; there is no need to clean them.
+
+    ![Screenshot of metadata manifest .csv file](./images/3_examine_csv.png)
+
+    > ## As an aside
```

Comment:
I do like this aside and the indentation with ">", however in the markdown preview it appears greyed out, including the image. Maybe this section could be surrounded by flanking "---" lines above and below it, signalling that it is an aside, without greying out the text and image?

Reply:
Good suggestion: implemented.